### PR TITLE
Integrate slackin into the website itself

### PR DESCRIPTION
**Pull request by pior**
# Why
Stop paying for a 99.9% inactive heroku app: http://slack.mtlpy.org/

# How
- Integrate our fork of [django-slackin](https://github.com/mtlpy/django-slackin)
- [x] set the slack api-key in Heroku envvars

Notes:
- this PR *doesn't* change the link in the footer.
- new link will be: `mtlpy.org/slackin`. 

Left after this PR:
- Redirect from http://slack.mtlpy.org/
- Change the link in the footer

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -46,3 +46,4 @@ jobs:
             AWS_ACCESS_KEY_ID: TEST_AWS_ACCESS_KEY_ID
             AWS_SECRET_ACCESS_KEY: TEST_AWS_SECRET_ACCESS_KEY
             AWS_STORAGE_BUCKET_NAME: TEST_AWS_STORAGE_BUCKET_NAME
+            SLACKIN_TOKEN: TEST_TOKEN
```
Comment (mlhamel) [inline .circleci/config.yml]: `TEST_SLACKIN_TOKEN` 🚲 🏠

File: Pipfile [hunk 2]
```diff
@@ -27,12 +27,15 @@ requests = "==2.18.0"
 sorl-thumbnail = "==12.3"
 python-dateutil = "*"
 futures = "*"
+"bfb3076" = {file = "https://github.com/mtlpy/django-slackin/archive/make-it-work.zip"}
```
Comment (mlhamel) [inline Pipfile]: wat ?

Comment (pior) [reply Pipfile]: Tu n'aimes pas ce petit côté *scrappy*? :trollface:

**Comment by mlhamel** [comment]
🚢

File: Pipfile (hunk 2 above)
Comment (pior) [reply Pipfile]: Seriously, I want to see if this package is good enough for us. If it is, I will put a bit more effort into it (like publish it to pypi)

Comment (mlhamel) [reply Pipfile]: hahaha on devrait au moins renomer en `bfg9000`

**Pull request merged**